### PR TITLE
Fons's Patches for macOS 11 Big Sur

### PR DESCRIPTION
**Pull request by Axel-Naumann**

### CODE REVIEW

File: test/Makefile [hunk 1]
```diff
@@ -605,8 +601,8 @@ $(STRESSHEPIX): $(STRESSHEPIXO) $(STRESSGEOMETRY) $(STRESSFIT) $(STRESSL) \
 		$(MT_EXE)
 		@echo "$@ done"
 
-$(STRESSROOFITO): stressRooFit_tests.cxx
-$(STRESSROOSTATSO): stressRooStats_tests.cxx stressRooStats_models.cxx
+$(STRESSROOFITO): stressRooFit_tests.h
+$(STRESSROOSTATSO): stressRooStats_tests.h stressRooStats_models.h
```
Comment (pcanal) [inline test/Makefile:605]: This does not seem directly related to the commit log "HASTHREAD" is gone.  If this change is needed, should line 606 also be changed and/or it should list both the .h and .cxx

Comment (Axel-Naumann) [reply test/Makefile:605]: @FonsRademakers can you comment?

Comment (FonsRademakers) [reply test/Makefile:605]: only the stressRooXXXX files were renamed from .cxx to .h by Lorenzo, there are no .cxx versions anymore. stressHistXXXX remained unchanged. Has nothing the do with HASTHREAD, if forgot to explicitly mention this file name change in the log message.

Comment (pcanal) [reply test/Makefile:605]: Could you update the commit log accordingly?  Thanks.

Comment (Axel-Naumann) [reply test/Makefile:605]: Done! (Split into separate commit.)

**Pull request merged**